client/rpc/validators.go

```diff
@@ -14,7 +14,8 @@ import (
 
 // TODO these next two functions feel kinda hacky based on their placement
 
-func validatorCommand() *cobra.Command {
+//ValidatorCommand returns the validator set for a given height
+func ValidatorCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "validatorset [height]",
 		Short: "Get the full validator set at given height",
```

Comment:
Also `validatorset` is the tendermint validator set - I think we somehow specify this more explicitly in the comments